components/compliance-service/inspec-agent/resolver/resolver.go

```diff
@@ -776,7 +776,9 @@ func convertNodeTcToInspecTc(tc *nodes.TargetConfig) (inspec.TargetConfig, error
 
 func (r *Resolver) resolveStaticJobInfo(job *jobs.Job, node *nodes.Node, tc inspec.TargetConfig, id string) types.InspecJob {
 	env := pgdb.FindKeyValue(node.Tags, "environment").Value
-
+	if len(env) == 0 {
+		env = pgdb.FindKeyValue(node.Tags, "Environment").Value
```

Comment:
we add this environment value to the node info, and that gets added to the report before ingestion